Library/Homebrew/tap.sh

```diff
@@ -0,0 +1,41 @@
+# Does the quickest output of brew tap possible for no arguments.
+# HOMEBREW_LIBRARY is set by bin/brew
+# shellcheck disable=SC2154
+
+normalise_tap_name() {
+  local dir="$1"
+  local user
+  local repo
+
+  user="$(echo "${dir%%/*}" | tr '[:upper:]' '[:lower:]')"
+  repo="$(echo "${dir#*/}" | tr '[:upper:]' '[:lower:]')"
```

Comment:
```suggestion
  user="$(tr '[:upper:]' '[:lower:]' <<<"${dir%%/*}")"
  repo="$(tr '[:upper:]' '[:lower:]' <<<"${dir#*/}")"
```
Maybe? Tho probably doesn't matter much.

Reply:
Thanks for reminding me again of this feature 😂